plugins/metabase-fetcher.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+import consola from 'consola'
+
+async function _getMetabaseSessionId(apiUrl, credentials) {
+  const sessionUrl = `${apiUrl}/api/session`
+  const headers = { 'Content-Type': 'application/json' }
+  const response = await axios.post(sessionUrl, credentials, { headers })
+  return response.data.id
+}
+
+export default async function () {
+  const apiUrl = process.env.METABASE_API_URL
+  const credentials = {
+    password: process.env.METABASE_PASSWORD,
+    username: process.env.METABASE_USERNAME,
+  }
+  if (!apiUrl || !credentials.password || !credentials.username) {
+    consola.warn('Metabase environments variable must be provided')
```

Comment:
Question: Pourquoi passer par une librairie plutôt que par le log classique ?

```suggestion
    consola.warn('Metabase environment variables must be provided')
```

Reply:
Je sais pas j'ai suivi ce qui a été fait dans le module `assets-extractor`.